backend/app/controllers/tasks.js

```diff
@@ -350,3 +375,73 @@ function* checkTaskData(req) {
         yield * common.checkDuplicateTask(req, req.body.stepId, req.body.uoaId, req.body.productId);
     }
 }
+
+function* updateCurrentStepId(req, insertedTaskId) {
+    var thunkQuery = req.thunkQuery;
+
+    // var essenceId = yield * getEssenceId(req, 'Tasks');
+    var product = yield * common.getEntity(req, req.body.productId, Product, 'id');
+
+    //TODO: Get survey from survery service if needed
+    // var survey = yield * common.getEntity(req, product.surveyId, Survey, 'id');
```

Comment:
Should we delete these comments? If the need arises later, we will probably know where it needs to go.

Reply:
Might as well.